{"name": "test_add_member_fail", "status": "passed", "description": "\n        当手机号码格式输入错误，有没有错误提示\n        :return:\n        ", "attachments": [{"name": "log", "source": "b1aacdfa-ac6d-4c88-81a5-ba805c26562a-attachment.txt", "type": "text/plain"}], "start": 1646663267408, "stop": 1646663268732, "uuid": "9bf7f137-4512-4f63-a760-440617993e92", "historyId": "97c580b3e694d82397d5aaf187803dfd", "testCaseId": "7f41945d287e68c4e42f02f0d8ae0790", "fullName": "case.test_addmember.TestAddMember#test_add_member_fail", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_addmember"}, {"name": "subSuite", "value": "TestAddMember"}, {"name": "host", "value": "DESKTOP-7NK0LBP"}, {"name": "thread", "value": "19400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_addmember"}]}